readthedocs/doc_builder/exceptions.py

```diff
@@ -57,13 +57,6 @@ class BuildUserError(BuildBaseException):
         "Ensure your project is configured to use the output path "
         "'$READTHEDOCS_OUTPUT/html' instead."
     )
-    PDF_COMMAND_FAILED = gettext_noop(
-        "PDF generation failed. "
-        "The build log below contains information on what errors caused the failure. "
-        "Our code has recently changed to fail the entire build on PDF errors, "
-        "where we used to pass the build when a PDF was created. "
-        "Please contact us if you need help understanding this error."
-    )
```

Comment:
Ah! Once we re-revert, this would be a great place to add helpful tips on how to get it working again.